modules/tinymce/src/plugins/accordion/test/ts/browser/QuirksTest.ts

```diff
@@ -31,4 +32,14 @@ describe('browser.tinymce.plugins.accordion.QuirksTest', () => {
     editor.dispatch('click', event );
     TinyAssertions.assertCursor(editor, [ 1, 0 ], 0);
   });
+
+  it('TINY-11753: pressing enter when a figure is selected should move teh selection in the text into the figcaption', () => {
+    const editor = hook.editor();
+    editor.setContent('<figure contenteditable="false"><img src="file.png"><figcaption contenteditable="true">Caption</figcaption></figure>');
+    TinySelections.select(editor, 'figure', []);
+    TinyAssertions.assertSelection(editor, [], 0, [], 1);
+
+    TinyContentActions.keystroke(editor, Keys.enter());
+    TinyAssertions.assertSelection(editor, [ 0, 1, 0 ], 0, [ 0, 1, 0 ], 0);
```

Comment:
If both paths and points are the same you can use `assertCursor` instead.

Reply:
changed